Firebase/Core/FIRApp.m

```diff
@@ -400,6 +401,41 @@ + (BOOL)isDefaultAppConfigured {
   return (sDefaultApp != nil);
 }
 
++ (void)registerLibrary:(nonnull NSString *)library withVersion:(nonnull NSString *)version {
+  // Create the set of characters which aren't allowed, only if this feature is used.
+  static NSCharacterSet *disallowedSet;
```

Comment:
Since this will likely only be used once per launch and not used after, does it make sense to store them as static vars? Would be nice to leave the memory footprint smaller if possible. I realize more than one library could be used but I don't think that'll happen too often.

Reply:
Sure, changed these to local vars.